dotnet/src/Connectors/Connectors.Memory.DuckDB/DuckDBExtensions.cs

```diff
@@ -17,4 +17,16 @@ public static float GetFloat(this DbDataReader reader, string fieldName)
         int ordinal = reader.GetOrdinal(fieldName);
         return reader.GetFloat(ordinal);
     }
+
+    public static object GetValue(this DbDataReader reader, string fieldName)
```

Comment:
I would make this method generic and call `GetFieldValue<>` so that you don't accidentally use boxing when reading primitive types.

Reply:
Done!